app/src/main/java/com/example/rum8/dataModels/LinkListSingleLink.java

```diff
@@ -37,4 +37,8 @@ public String getUid() {
         return this.uid;
     }
 
+    public void setBitMap(Bitmap bitMap){
```

Comment:
Do you think we should name this something less technical? Like `setPicture` or `setImage`?